backend/src/clients/s3.ts

```diff
@@ -31,3 +31,21 @@ export async function putObjectStream(bucket: string, key: string, body: Readabl
     fileSize,
   }
 }
+
+export async function getObjectStream(bucket: string, key: string, range?: { start: number; end: number }) {
+  const client = await getS3Client()
```

Comment:
Do we need to create a new S3 Client for every S3 request?

Reply:
Our S3 credentials expire on some platforms relatively frequently.  Tracking whether the current S3 client has expired and replacing it seems too complex for a function that's likely to only get called a few hundred - thousand times a day.